javascript/javascript1/week1/main.js

```diff
@@ -0,0 +1,46 @@
+/*1. Future age calculator*/
+let yearOfBirth = 1989;
+let yearFuture = 2045;
+let age = yearFuture - yearOfBirth;
+console.log('Your age will be either ' + age + ' or ' + (age - 1));
+
+/*2. Dog age calculator*/
+
+let dogYearOfBirth = 2019;
+let dogYearFuture = 2027;
+let humanYear = dogYearFuture - dogYearOfBirth;
+let shouldShowResultInDogYears = 7 * humanYear;
+
+if (shouldShowResultsInDogYears = true) {
+        console.log(`Your dog age will be ${shouldShowResultInDogYears} dog years old in 2027.`);
```

Comment:
here this is not correct. you should have `${dogyears}` and  you should probably declare something like this `let dogyears = 7 * humanYear `